account_invoice_view_payment/tests/test_account_invoice_view_payment.py

```diff
@@ -143,7 +143,6 @@ def test_view_account_payment_register_form(self):
         ).create(
             {
                 "journal_id": self.cash.id,
-                "payment_method_id": self.payment_method_manual_in.id,
```

Comment:
why?

Reply:
I thought it was required, I'll remove it